API/API.mli

```diff
@@ -117,11 +116,6 @@ sig
 
   val enforce_leq : Universe.t constraint_function
   val pr_uni : Universe.t -> Pp.std_ppcmds
-  val pr_universe_context : (Level.t -> Pp.std_ppcmds) -> UContext.t -> Pp.std_ppcmds
-  val pr_universe_context_set : (Level.t -> Pp.std_ppcmds) -> ContextSet.t -> Pp.std_ppcmds
-  val pr_universe_subst : universe_subst -> Pp.std_ppcmds
-  val pr_universe_level_subst : universe_level_subst -> Pp.std_ppcmds
-  val pr_constraints : (Level.t -> Pp.std_ppcmds) -> Constraint.t -> Pp.std_ppcmds
```

Comment:
It is interesting that these functions are not used by our plugins. I agree that keeping them violates the declared [API/PROPERITES](https://github.com/coq/coq/blob/master/API/PROPERTIES). However, in this case I think it is `API/PROPERTIES` that need to be updated instead of removing these functions (in order to achieve consistency between the actual code and `API/PROPERTIES` file).

Updates of the `API/PROPERTIES` file is part of [the plan](https://github.com/matejkosik/coq/blob/trunk%2BAPI_ROADMAP/API/ROADMAP).